### PR TITLE
Pushing codeblock to the side

### PR DESCRIPTION
**Pull request by tahmidefaz**
Using Flex layout to push codeblocks to the side.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "apidocs",
   "version": "0.1.0",
   "private": true,
-  "homepage": "apidocs",
+  "homepage": ".",
```
Comment (josejulio) [inline package.json:5]: won't this break the deployment? 
react uses the `homepage` thing to setup the base-url for all the assets.  
Using `.` will make the assets relative to the `index.html`

File: src/App.scss [hunk 2]
```diff
@@ -1 +0,0 @@
-
```
Comment (josejulio) [inline src/App.scss:1]: @epwinchell added this file IIRC - not sure if it's a placeholder or if is actually needed to load `scss` from patternfly.

File: package.json (hunk 1 above)
Comment (tahmidefaz) [reply package.json]: Removed this change :+1:

File: src/App.scss (hunk 2 above)
Comment (tahmidefaz) [reply src/App.scss]: Not sure, maybe a placeholder. I stashed the line change on that file.

**Review by josejulio — APPROVED** [review]

**Pull request merged**